cumulus/parachains/integration-tests/emulated/tests/people/people-rococo/src/tests/reap_identity.rs

```diff
@@ -0,0 +1,496 @@
+		),
+		(
+			Data::Raw(b"discord".to_vec().try_into().unwrap()),
+			Data::Raw(b"bohr-username".to_vec().try_into().unwrap()),
+		),
+	])
+	.unwrap();
+	let mut fourth_para = second_para.clone();
+	fourth_para.github = Data::Raw(b"niels-username".to_vec().try_into().unwrap());
+	fourth_para.discord = Data::Raw(b"bohr-username".to_vec().try_into().unwrap());
+
+	vec![
+		Identity {
+			relay: RelayIdentity { main: first_relay, subs: Subs::One },
+			para: ParaIdentity { main: first_para, subs: Subs::One },
+		},
+		Identity {
+			relay: RelayIdentity { main: second_relay, subs: Subs::One },
+			para: ParaIdentity { main: second_para.clone(), subs: Subs::One },
+		},
+		Identity {
+			relay: RelayIdentity { main: third_relay, subs: Subs::One },
+			para: ParaIdentity { main: second_para.clone(), subs: Subs::One }, /* same as
+			                                                                    * second_para */
+		},
+		Identity {
+			relay: RelayIdentity { main: fourth_relay.clone(), subs: Subs::One },
+			para: ParaIdentity { main: fourth_para.clone(), subs: Subs::One },
+		},
+		Identity {
+			relay: RelayIdentity { main: fourth_relay.clone(), subs: Subs::Many(2) }, // 2
+			// subs
+			para: ParaIdentity { main: fourth_para.clone(), subs: Subs::Many(2) },
+		},
+		Identity {
+			relay: RelayIdentity {
+				main: fourth_relay.clone(),
+				subs: Subs::Many(MaxSubAccounts::get()),
+			},
+			para: ParaIdentity {
+				main: fourth_para.clone(),
+				subs: Subs::Many(MaxSubAccounts::get()),
+			},
+		},
+		Identity {
+			relay: RelayIdentity { main: fourth_relay.clone(), subs: Subs::Zero },
+			para: ParaIdentity { main: fourth_para.clone(), subs: Subs::Zero },
+		},
+	]
+}
+
+fn id_deposit_parachain(id: &IdentityInfoParachain) -> Balance {
+	let base_deposit = BasicDepositParachain::get();
+	let byte_deposit =
+		ByteDepositParachain::get() * TryInto::<u128>::try_into(id.encoded_size()).unwrap();
+	base_deposit + byte_deposit
+}
+
+fn id_deposit_relaychain(id: &IdentityInfo<MaxAdditionalFields>) -> Balance {
+	let base_deposit = BasicDeposit::get();
+	let byte_deposit = ByteDeposit::get() * TryInto::<u128>::try_into(id.encoded_size()).unwrap();
+	base_deposit + byte_deposit
+}
+
+fn set_id_relay(id: &Identity) -> Balance {
+	let mut total_deposit = 0_u128;
+
+	// Set identity and Subs on Relay Chain
+	RococoRelay::execute_with(|| {
+		type RuntimeEvent = <RococoRelay as Chain>::RuntimeEvent;
+
+		assert_ok!(RococoIdentity::set_identity(
+			RococoOrigin::signed(RococoRelaySender::get()),
+			Box::new(id.relay.main.clone())
+		));
+
+		match id.relay.subs {
+			Subs::Zero => {},
+			Subs::One => {
+				assert_ok!(RococoIdentity::set_subs(
+					RococoOrigin::signed(RococoRelaySender::get()),
+					vec![(
+						RococoRelayReceiver::get(),
+						Data::Raw(vec![1_u8; 1].try_into().unwrap())
+					)],
+				));
+			},
+			Subs::Many(n) => {
+				let mut subs = Vec::new();
+				for i in 0..n {
+					subs.push((
+						RococoRelayReceiver::get(),
```

Comment:
Should not all be the same account.

Reply:
Yeah, hmm.

In the case of `MaxSubAccounts` which is 100, do you know of anyway i can dynamically create n number of accounts? If there is no way, I think this would be nice to add to xcm-emulator.

Reply:
I can use `AccountId32::new()` 👍